### ISSUE TITLE
support for defining some of the env variables in the credentials section of huginn

### ISSUE THREAD

**Issue by mogorman** [cantino/huginn#1675]
feature request:  I would like to define smtp settings in the app rather than in the env file. this would be nice as it would allow for more accounts to be used from one instances as well as make it easier to make configuration changes over time.

I think this also makes sense for all the oauth keys and secrets as well.

**Comment by dsander** [comment]
Yes that would be very nice. The issue with storing the configuration in the database is that we can't efficiently cache it. So Huginn would either require a restart after the configuration was modified, or every access of a setting would trigger a database query. 
I am pretty sure we had a discussed about it in a different issue already, but I could not find it :(
